### ISSUE TITLE
content: obsolete screen shot JWSzKy951NpiznLGxqoQ.png

### ISSUE THREAD

**Issue by yecril71pl** [GoogleChrome/web.dev#7385]
### What page(s) need to be updated?

![obsolete screen shot](https://web-dev.imgix.net/image/tcFciHGuF3MxnTr1y5ue01OGLBn2/JWSzKy951NpiznLGxqoQ.png)

### Why is this update needed?

https://github.com/GoogleChrome/lighthouse/pull/13517

**Comment by heyawhite** [comment]
@yecril71pl can you point us to where that screenshot is from on web.dev or developer.chrome.com? Or if it's not on those sites, this issue will need to be opened in the appropriate repository.

Thanks so much

**Comment by yecril71pl** [comment]
[Learn more](https://web.dev/object-alt/)

**Comment by yecril71pl** [comment]
It does not look like it is going to be removed any time soon, especially given that the alternative document is wrong.  This is a trivial clerical task that does not seem would take 2 months to accomplish.  However, it requires special privileges.  Please reopen?

**Comment by jeffposnick** [comment]
Once https://github.com/GoogleChrome/web.dev/pull/7404 is merged, requests for https://web.dev/object-alt/ will redirect to https://dequeuniversity.com/rules/axe/4.4/object-alt

If you;re saying that there's also an error in https://dequeuniversity.com/rules/axe/4.4/object-alt, I would consider following up with the folks responsible for that content, perhaps via https://github.com/dequelabs/axe-core#contributing

**Comment by yecril71pl** [comment]
> Once #7404 is merged, requests for https://web.dev/object-alt/ will redirect to https://dequeuniversity.com/rules/axe/4.4/object-alt
> 
> If you;re saying that there's also an error in https://dequeuniversity.com/rules/axe/4.4/object-alt, I would consider following up with the folks responsible for that content, perhaps via https://github.com/dequelabs/axe-core#contributing

The question is, why I should be the person to talk to Deque.  I have no business with them and it is not my decision to redirect.  It looks like a whack-a-mole game now.

Of course, the situation will be different when the move has been done, even though it should not be, which will put me into damage control mode.  But it is not being done for a long time now and therefore it seems reasonable to fix our documentation nevertheless.  You could have done this 100&times; by now.

**Comment by rachelandrew** [comment]
@yecril71pl I'm the content lead for web.dev and developer.chrome.com and the person who closed the issue.

Our small team have lots of things we need to work on, so we have to prioritise. Updating a piece of content that is going to go away, is not a priority, when we have plenty of things that need updating that are staying.

You are very welcome to open a PR to fix the issue, and assuming it's a straightforward image replacement I'll happily approve it, however this content is being removed and redirected, so it's up to you if you feel it is worth the time. 

I understand this is a problem to you, however the decision to move this content has been made, so if you do feel the document at Deque is incorrect it would be most useful to go and raise an issue over there.

**Comment by yecril71pl** [comment]
> You are very welcome to open a PR to fix the issue, and assuming it's a straightforward image replacement I'll happily approve it, however this content is being removed and redirected, so it's up to you if you feel it is worth the time.

I do not think the screen shot can be fixed using a PR.  Of course, I can replace the screen shot URL, but there is no suitable URL to use and AIUI there is no way for a community contributor to push a new image onto the image server.

> 
> I understand this is a problem to you, however the decision to move this content has been made, so if you do feel the document at Deque is incorrect it would be most useful to go and raise an issue over there.

As I have said, I hope you will do it because you have no interest in degrading the content; if you fail, I shall see to it after the damage has been done.
